data.sql

```diff
@@ -0,0 +1,11 @@
+vet_clinic=# INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
```

Comment:
- [ ] Kindly add the missing things in your readme file such as **description**, **languages and technologies**, **authors**, **issues page link**, **acknowledgment** and **license** :blush: You should use [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) for making an descriptive readme file :+1: